### PR TITLE
Knesset Openness dial

### PR DESCRIPTION
**Pull request by daonb**
A new app for displaying an openness dial (actually supporting many dials).
The slug, grade and description are set in the admin interface and the result is a view http://oknesset.org/<slug> that shows the dial and includes a link to a nice description page.

#759

### CODE REVIEW

File: templates/dials/dial.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
```
Comment (alonisser) [inline templates/dials/dial.html]: How would this be embedded in other sites? Iframe? other?

File: templates/dials/dial.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+<html lang="he">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Language" content="he">
+    <title>כמה הכנסת פתוחה?</title>
+</head>
+<body>
+<a href="{% url 'dial-desc' slug %}" target="_blank">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 1097 479" enable-background="new 0 0 1097 479" xml:space="preserve">
+<g id="Layer_2">
```
Comment (alonisser) [inline templates/dials/dial.html]: SRSLY? A completely manual svg? no framework to generate it? no higher abstraction? something that no one could ever change or fix?

**Comment by daonb** [comment]
There's something missing - adding this to the home page. 

Oh, well, it's a new issue #771

File: templates/dials/dial.html (hunk 2 above)
Comment (daonb) [inline templates/dials/dial.html]: I've added the original file under `dials/svg`.

File: dials/tests.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (alonisser) [inline dials/tests.py]: @daonb please add a functional test for the view.. we don't add any more code to Open Knesset without tests. the era has ended..

File: dials/views.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+def dial_svg(request, slug):
+    dial = get_object_or_404(Dial, slug=slug)
+    if dial.precent < 15:
+        color = "#ED1C24"
+    elif dial.precent < 40:
+        color = "orange" #TODO: pick a color
+    else:
+        color = "green" #TODO: pick a color
+    return render_to_response("dials/dial.html",
+            {'width': dial.precent * 746 / 100,
```
Comment (alonisser) [inline dials/views.py]: @daonb Please change "magic numbers" to named constants, also is this always the same width? in mobile screens for example?

File: templates/dials/dial.html (hunk 2 above)
Comment (alonisser) [inline templates/dials/dial.html]: :+1:

File: dials/tests.py (hunk 3 above)
Comment (alonisser) [inline dials/tests.py]: @daonb please add a functional test for the view.. we don't add any more code to Open Knesset without tests. the era has ended..

File: dials/views.py (hunk 4 above)
Comment (alonisser) [inline dials/views.py]: @daonb Please change "magic numbers" to named constants, also is this always the same width? in mobile screens for example?

**Comment by alonisser** [comment]
@daonb See my latest comments and requests for changes. also please rebase from upstream/master so the code in the PR would be updated

File: dials/tests.py (hunk 3 above)
Comment (daonb) [inline dials/tests.py]: I'm glad we're done with that era :-)
Seriously, it's all boiler plate code and templates. I don't see what we can test here.

File: dials/views.py (hunk 4 above)
Comment (daonb) [inline dials/views.py]: It's SVG and it just scales.

**Pull request merged**